### PR TITLE
Server and script

### PR DESCRIPTION
**Pull request by SlingMaster**

### CODE REVIEW

File: app/src/main/java/com/jsc/smartpanel/SettingsActivity.java [hunk 1]
```diff
@@ -197,7 +198,7 @@ public void onCreate(Bundle savedInstanceState) {
             // to their values. When their values change, their summaries are
             // updated to reflect the new value, per the Android Design
             // guidelines.
-            bindPreferenceSummaryToValue(findPreference("server_port"));
+            //  bindPreferenceSummaryToValue(findPreference("server_port"));
```
Comment (uchitel) [inline app/src/main/java/com/jsc/smartpanel/SettingsActivity.java:201]: Udali stroku

File: app/src/main/java/utils/SysUtils.java [hunk 2]
```diff
@@ -85,6 +85,47 @@ public static void restartApp(Activity mainActivity, String nextApp, boolean nex
...
+//        // create a GregorianCalendar with the Pacific Daylight time zone
+//        // and the current date and time
+//        Calendar calendar = new GregorianCalendar(pdt);
+//        Date curTime = new Date();
+//        calendar.setTime(curTime);
+        // int hh = calendar.get(Calendar.HOUR);
+
+//        Calendar calendar = Calendar.getInstance();
+//        calendar.set(Calendar.HOUR_OF_DAY, 24);
+//        int hh = calendar.get(Calendar.HOUR);
```
Comment (uchitel) [inline app/src/main/java/utils/SysUtils.java:115]: Udali vse chto vishe

File: app/src/main/java/utils/GlobalUtils.java [hunk 3]
```diff
@@ -19,7 +19,11 @@
 import java.net.InetAddress;
 import java.net.NetworkInterface;
 import java.net.SocketException;
+import java.text.SimpleDateFormat;
+import java.util.Calendar;
+import java.util.Date;
 import java.util.Enumeration;
+import java.util.Locale;
```
Comment (uchitel) [inline app/src/main/java/utils/GlobalUtils.java:26]: Udali vse chto zakomentirovanno v etom faile

File: app/src/main/java/com/jsc/smartpanel/FullscreenActivity.java [hunk 4]
```diff
@@ -659,42 +674,83 @@ public void onBackPressed() {
...
+        SysUtils.setBackLight(this, night);
+    }
+
+    // ===================================
+    // Swap Screen Timer
+    // ===================================
+
+    // ===================================
+    private void startTimer() {
+        if (preference.getBoolean("sw_swap", true)) {
```
Comment (uchitel) [inline app/src/main/java/com/jsc/smartpanel/FullscreenActivity.java:712]: Prover' ili timer uzhe zapuschen.

File: app/src/main/java/com/jsc/smartpanel/FullscreenActivity.java [hunk 5]
```diff
@@ -280,50 +291,53 @@ public void webViewEvents(int request, final String jsonString) {
...
+//////                            external_cmd = Constants.CMD_LOAD_WEATHER;
+////                            external_cmd = Constants.CMD_LOAD_SMART;
+////                            break;
+////                        default:
+////                            external_cmd = 0;
+////                            break;
+////                    }
+////                    System.out.println("trace   | external_cmd = " + external_cmd + " | jsonString:" + jsonString);
+//////                    return;
+////                }
```
Comment (uchitel) [inline app/src/main/java/com/jsc/smartpanel/FullscreenActivity.java:325]: Udali comentarij

**Comment by SlingMaster** [comment]
remove old request and added new

**Pull request closed without merging**